ideascube/models.py

```diff
@@ -102,6 +115,17 @@ def has_perm(self, perm, obj=None):
     def has_module_perms(self, app_label):
         return True
 
+    def validate_unique(self, exclude=None):
+        super().validate_unique(exclude=exclude)
+
+        if self.id is None:
+            # We are creating a new user
+            same = User.objects.all(really=True).filter(serial=self.serial)
```

Comment:
I'd have used the `.exists()` method: https://docs.djangoproject.com/en/1.10/ref/models/querysets/#django.db.models.query.QuerySet.exists